weldx/transformations/local_cs.py

```diff
@@ -292,8 +296,20 @@ def _build_coordinates(
         if not isinstance(coordinates, xr.DataArray):
             if not isinstance(coordinates, (np.ndarray, pint.Quantity)):
                 coordinates = np.array(coordinates)
+
             coordinates = ut.xr_3d_vector(coordinates, time)
 
+        if isinstance(coordinates.data, pint.Quantity):
+            # The first branch is a workaround until units are mandatory
+            if coordinates.data.u == pint.Unit(""):
+                coordinates.data = coordinates.data.m
```

Comment:
This looks dangerous to me, I guess the assumption is that if a 'dimensionless' quantity is passed we interpret it as 'default unit' ?
I would prefer to be strict here and force all quantities to be `[length]` so that at least the quantity behavior is already consistent

Reply:
The problem is that our "unitless" code breaks at multiple places when we remove this line. This is just a workaround to enable units as input without enforcing them. For example, the "TimeSeries" returns unitless quantities in several tests and examples/notebooks. Fixing those problems is part of the transitions second step.

Reply:
ah I see, thank you
Is it only tests/notebooks or also inside the package?